core/services/request/files/FileSubmission.php

```diff
@@ -0,0 +1,182 @@
+<?php
```

Comment:
this is part of https://github.com/eventespresso/event-espresso-core/pull/999 which was already reviewed (just haven't gotten around to merging it)